base_classes/nyaml/NXinstrument.yaml

```diff
@@ -35,6 +35,7 @@ NXinstrument(NXobject):
   (NXfilter):
   (NXflipper):
   (NXguide):
+  (NXhistory):
```

Comment:
Ideally, this would be part of every subclass as well, no? Like now, there is only one history for the whole experiment, and not e.g. specific to an NXspin_filter etc.

Reply:
I think that this is what we should eventually come up with. But as we discussed I think we agreed that we will put it just for `NXinstrument` and add it later in a more generic base class for a device to not repeat everything.